bindings/python/client.py

```diff
@@ -225,13 +227,13 @@ def main():
     if options.max_download_rate <= 0:
         options.max_download_rate = -1
 
-    settings = { 'user_agent': 'python_client/' + lt.__version__,
-        'listen_interfaces': '%s:%d' % (options.listen_interface, options.port),
-        'download_rate_limit': int(options.max_download_rate),
-        'upload_rate_limit': int(options.max_upload_rate),
-        'alert_mask': lt.alert.category_t.all_categories,
-        'outgoing_interfaces' : options.outgoing_interface
-    }
+    settings = {'user_agent': 'python_client/' + lt.__version__,
+                'listen_interfaces': '%s:%d' % (options.listen_interface, options.port),
+                'download_rate_limit': int(options.max_download_rate),
+                'upload_rate_limit': int(options.max_upload_rate),
+                'alert_mask': lt.alert.category_t.all_categories,
+                'outgoing_interfaces': options.outgoing_interface
+                }
```

Comment:
this formatting is enforced by flake8?

Reply:
It looks like because the original statement split after the first line, the next lines were indented to align with the {. If we move the first item to a new line then we get something like this:

```python
    settings = {                                  
        'user_agent': 'python_client/' + lt.__version__,                                                              
        'listen_interfaces': '%s:%d' % (options.listen_interface, options.port),
        'download_rate_limit': int(options.max_download_rate),    
        'upload_rate_limit': int(options.max_upload_rate),                                                            
        'alert_mask': lt.alert.category_t.all_categories,           
        'outgoing_interfaces': options.outgoing_interface,               
    }                                                               
```
This passes the linter and looks better to me.